src/responses.rs

```diff
@@ -139,13 +151,15 @@ pub struct GetLog {
 }
 
 /// 'listconfigs' command
-pub type ListConfigs = HashMap<String, Json>;
+pub type ListConfigs = HashMap<String, serde_json::Value>;
 
 /// Sub-structure for channel in 'listpeers'
 #[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct Channel {
     pub state: String,
+    #[serde(default)]
```

Comment:
I think these are redundant. At least I never had to put them and never had an issue.

Reply:
I couldn't find documentation on it, so I just added it. Can test it without if you want.

Reply:
You were right: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=d2ea872eb983514df2a34396357dfff4